### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by jodator**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Improve feature documentation.

---

### CODE REVIEW

**Review by mlewand — APPROVED** [review]
LGTM, added one minor correction.

**Comment by mlewand** [comment]
@AnnaTomanek please have a look at it.

File: src/indentblockcommand.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ export default class IndentBlockCommand extends Command {
  */
 
 /**
- * Returns new indent attribute value based on current indent.
+ * Returns new indent attribute value based on current indent. Method returns `undefined` when indentation should be removed.
```
Comment (AnnaTomanek) [inline src/indentblockcommand.js]:
```suggestion
 * Returns a new indent attribute value based on the current indent. This method returns `undefined` when the indentation should be removed.
```

File: src/indentblockcommand.js [hunk 2]
```diff
@@ -105,7 +105,7 @@ export default class IndentBlockCommand extends Command {
...
  *
  * @method #getNextIndent
  * @param {String} indentAttributeValue Current indent attribute value.
```
Comment (AnnaTomanek) [inline src/indentblockcommand.js]:
```suggestion
 * @param {String} indentAttributeValue The current indent attribute value.
```

File: src/indentcommandbehavior/indentusingclasses.js [hunk 3]
```diff
@@ -14,7 +14,7 @@
  */
 export default class IndentUsingClasses {
 	/**
-	 * Creates an instance of the command.
+	 * Creates an instance of the behavior.
```
Comment (AnnaTomanek) [inline src/indentcommandbehavior/indentusingclasses.js]:
```suggestion
	 * Creates an instance of the indentation behavior.
```

File: src/indentediting.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ import MultiCommand from '@ckeditor/ckeditor5-core/src/multicommand';
  * This plugin registers the `'indent'` and `'outdent'` commands.
  *
  * **Note**: In order the commands to work at least one of compatible features is required. Read more in
```
Comment (AnnaTomanek) [inline src/indentediting.js]:
```suggestion
 * **Note**: In order for the commands to work, at least one of the compatible features is required. Read more in:
```

**Review by AnnaTomanek — CHANGES_REQUESTED** [review]

**Pull request merged**